docker-compose.yaml

```diff
@@ -106,8 +106,8 @@ services:
       - AWS_ENDPOINT_URL=http://localhost:4566
       - S3_BUCKET=lumigator-storage
       - PYTHONPATH=/mzai/lumigator/python/mzai/backend
-      - EVALUATOR_PIP_REQS=/mzai/lumigator/python/mzai/jobs/evaluator/requirements.txt
-      - EVALUATOR_WORK_DIR=/mzai/lumigator/python/mzai/jobs/evaluator
+      - EVALUATOR_PIP_REQS=/mzai/lumigator/python/mzai/jobs/eval_lite/requirements.txt
```

Comment:
@aittalam fyi heads up here's the scope of work I've been doing that parallels yours. Not ready yet, just an idea of where I'm headed

Reply:
Will we have a period in which both the old and the new evals run simultaneously (e.g. for testing one against the other)? If so, perhaps we could add a specific workdir/requirements pair for eval_lite instead of just changing the existing one.

Reply:
WDYT makes more sense? I was thinking about it as: 
1. This PR gets approved, which works with eval lite
2. I cut a branch off this PR which approves eval
3. Both are merged together which means new eval runs off of eval-lite that's renamed. 

I was hoping to avoid having both at once and get rid of the old evaluator code but I'm not sure what the global impact will be

Reply:
Yeah, one thing I am a bit afraid of is the need for both inference and lite_eval to be up and running PLUS the extra code to deal with the multi-step process before being able to substitute current evaluation. This would make it a bit harder for us to do things incrementally, especially given the quickly-moving codebase.

Reply:
Let's discuss this particular piece in the context of experiments and where we're going with the jobs 😄

Reply:
keeping it separate for now